core/modules/replica/DatabaseMySQLExceptions.h

```diff
@@ -55,14 +55,8 @@ namespace mysql {
  */
 class Error : public std::runtime_error {
 public:
-
-    /**
-     * @param what
-     *   reason for the exception
-     */
-    explicit Error(std::string const& what)
-        :   std::runtime_error(what) {
-    }
+    /// @param what  reason for the exception
+    explicit Error(std::string const& what) : std::runtime_error(what) {}
```

Comment:
You can avoid writing this by using inherited constructor: `using std::runtime_error::runtime_error`.

Reply:
I wasn't aware about this trick! Thanks!

Reply:
Okay. I have tried this, and it's not going to work since this is the `C++17` feature as per:
https://en.cppreference.com/w/cpp/language/using_declaration
Though, I like the idea. And I will keep it in mind.

Reply:
Or, to me more precise - most probably it would work with out current "almost-17" compiler which includes many features of `c++17` as "the experimental support", but we're not supposed to use this feature for now.

Reply:
This is actually C++11 feature, should be safe to use.